migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,62 @@
+[id='Migration-toolkit-for-containers-overview']
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: Migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the MigrationToolkit for Containers (MTC). To know more about MTC see the xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[MTC installation]: You can install the Migration Toolkit for Containers (MTC) on {product-title} 4. After installing MTC, you must configure an object storage to use as a replication repository. You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You can do the following as a part of installing MTC:
+
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[Migration Toolkit for Containers Operator on {product-title}]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#migration-configuring-proxies_installing-mtc[Proxies]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#configuring-replication-repository_installing-mtc[Replication repository]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#migration-configuring-mcg_installing-mtc[Multicloud Object Gateway]
+
+You can xref:../migration_toolkit_for_containers/installing-mtc-restricted.adoc#installing-mtc-restricted[Install the MTC in a restricted network environment].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[Upgrade MTC]: You can upgrade the MTC on {product-title} 4.6 by using the Operator Lifecycle Manager.
```

Comment:
This is mentioning 4.6 - should it say 4.6 or later? Or not mention it at all?

Reply:
changed to not mentioning 4.6 at all